lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -203,6 +162,103 @@ sub _files_to_categories {
     return $ret;
 }
 
+my $rt_cpan_base = 'https://rt.cpan.org/Ticket/Display.html?id=';
```

Comment:
I was just curious.  I thought it might be easier to test in a model and I think it got pulled from a model into the controller.